### DISCUSSION TITLE
Stellarium Running Slow

### DISCUSSION THREAD

**Discussion by chisholmjohn** [Stellarium/stellarium#3022] [General]
Hello everyone, 

First off, I love Stellarium. I use it extensively in my Intro to Astronomy class at the University of Texas. We used it for the first time this past week during lecture and the students were experiencing extreme lag when they were using it (as in it would take 15-30 seconds to even move to a different location on the sky). This was on a wide variety of different computer tips (although Macs seemed to be the most common). I have used Stellarium for a few semesters now, and I have never encountered this problem. I was running an older version (0.21.3) and did not seem to have issues. Is this a current problem with newer versions of Stellarium, or are there other tips to help students with performance? I searched the discussion board, but I couldn't find any tips. Thanks for your help!

**Comment by alex-w** [comment]
macOS on Apple Silicon devices?

**Comment by chisholmjohn** [comment]
It was both. Some were quite old Macs and some were M1 chips.

**Reply by alex-w** [reply]
Please see #2836 and #2947 issues - something else?

**Comment by 10110111** [comment]
This might be due to #2778.
